resources/views/inputs/upload.blade.php

```diff
@@ -34,12 +34,12 @@ class="block absolute top-0 opacity-0 cursor-pointer"
                                 class="link"
                                 @if ($click ?? false) @click="{{ $click }}" @endif
                             >
-                                Upload a file
+                                @lang('ui::actions.upload_file')
                             </button>
                         </p>
 
                         <p class="mt-1 text-xs text-theme-secondary-500">
-                            {{ strtoupper(implode(', ', $extensions ?? [])) }} up to {{ $size ?? '10' }}MB
+                            {{ strtoupper(implode(', ', $extensions ?? [])) }} @lang('ui::generic.up_to') {{ $size ?? '10' }}@lang('ui::numeric.mb')
```

Comment:
wouldn't it be better to make this a single text with parameters? Using a lang file while hardcoding values like that defeats the purpose of having them as they can change between languages (e.g. $1.00 vs 1,00$ as example)